src/Symfony/Bundle/Resources/config/state/provider.xml

```diff
@@ -25,6 +25,11 @@
             <argument type="service" id="translator" on-invalid="null" />
         </service>
 
+        <service id="api_platform.state_provider.backed_enum" class="ApiPlatform\State\Provider\BackedEnumProvider">
+            <tag name="api_platform.state_provider" key="ApiPlatform\State\Provider\BackedEnumProvidevr" />
+            <tag name="api_platform.state_provider" key="api_platform.state_provider.backed_enum" />
+        </service>
```

Comment:
For the record @GwendolenLynch instead of decorating the provider chain, I choose to declare this as an operation provider instead. This will be called inside the ReadProvider: 

https://github.com/api-platform/core/blob/ef4812a688d4cd20877f5bad52d036e80cd43581/src/State/Provider/ReadProvider.php#L75-L79 

It's easier for userland to decorate our enum provider (or any item provider) instead of hooking into the main decoration chain.